src/main/java/org/broadinstitute/hellbender/engine/MultiVariantWalker.java

```diff
@@ -22,11 +20,12 @@
  */
 public abstract class MultiVariantWalker extends VariantWalkerBase {
 
-    // NOTE: using File rather than FeatureInput<VariantContext> here so that we can keep this driving source
-    //       of variants separate from any other potential sources of Features
-    @Argument(fullName = StandardArgumentDefinitions.VARIANT_LONG_NAME, shortName = StandardArgumentDefinitions.VARIANT_SHORT_NAME,
-                doc = "One or more VCF files containing variants", common = false, optional = false)
-    public List<String> drivingVariantFiles = new ArrayList<>();
+    @ArgumentCollection
+    protected MultiVariantInputArgumentCollection multiVariantInputArgumentCollection = getMultiVariantInputArgumentCollection();
+
+    protected MultiVariantInputArgumentCollection getMultiVariantInputArgumentCollection() {
```

Comment:
Minor nit request to move this down in the file past the variable declarations and into the section with the other methods, with a javadoc comment.

Reply:
ok